v2/awsv1shim/session.go

```diff
@@ -50,10 +50,10 @@ func getSessionOptions(awsC *awsv2.Config, c *awsbase.Config) (*session.Options,
 		},
 	}
 
-	if !c.SuppressDebugLog {
-		options.Config.LogLevel = aws.LogLevel(aws.LogDebugWithHTTPBody | aws.LogDebugWithRequestRetries | aws.LogDebugWithRequestErrors)
-		options.Config.Logger = debugLogger{}
-	}
+	// if !c.SuppressDebugLog {
+	// 	options.Config.LogLevel = aws.LogLevel(aws.LogDebugWithRequestRetries)
+	// 	// options.Config.Logger = debugLogger{}
+	// }
```

Comment:
Can this commented code be removed with the logic moving up into the `GetSession` func?